Update github issue templates (#2)
ansible-community/meetup: .github/ISSUE_TEMPLATE/submit-a-talk.md

---
name: Submit a talk
about: Tell us about a talk you'd like to give
body:
- type: markdown
  attributes:
    value: >
      Thank you for your interest in talking at an Ansible community meetup !
      We are looking forward to your proposal on a wide range of Ansible topics whether you are a user, a contributor or a developer.

>       If you have any questions, please feel free to reach out to us in #community:ansible.com on Matrix or #ansible-community on [libera.chat](https://libera.chat/).

Reviewer: ```suggestion
      If you have any questions, please feel free to reach out to us via:
      * Matrix: [#community:ansible.com](https://matrix.to/#/#community:ansible.im)
      * Libera.chat (IRC):` #ansible-community` on [libera.chat](https://web.libera.chat/#libera)
```
